app/src/main/java/com/github/freeman/bootcamp/auth/FirebaseAuthActivity.kt

```diff
@@ -57,12 +70,25 @@ class FirebaseAuthActivity : ComponentActivity() {
                 )
             }
         }
+
+
     }
 
+
     /**
-     * Deletes the google account from the device
+     * Deletes the 'Guess It!' account from the device
      */
-    fun deleteGoogleAccount() {
+    fun deleteAccount() {
```

Comment:
This function is a great addition as it deletes all the different elements that are linked with an account, good job !